ui/src/app/components/database-loader/database-loader.component.html

```diff
@@ -4,7 +4,12 @@
     alt="database-loader-gif"
     class="loader-gif"
   />
-  <h3 *ngIf="loaderType === 'direct'">Connecting to {{ databaseName }} database...</h3>
+  <h3 *ngIf="loaderType === 'direct'">Populating schema for {{ databaseName }} database. Please wait...</h3>
```

Comment:
Populating schema sounds misleading , should we say Reading infoschema for databse ...

Reply:
Done